wink/signal.hpp

```diff
@@ -67,7 +67,12 @@ namespace wink
 		template <typename... Args>
 		void disconnect(Args&&... args)
 		{
-			_slots.erase(std::find(_slots.begin(), _slots.end(), slot_type(std::forward<Args>(args)...)));
+			typename slot_array::const_iterator it = std::find(_slots.begin(), _slots.end(), slot_type(std::forward<Args>(args)...));
```

Comment:
Since I'm already using C++11 features (variadic templates), is there a reason why you don't just do `const auto = ...`?